custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
```

Comment:
To harmonize coding style, can you use VSCode with black code formatting at line-length 88?

I can show you how to set up VSCode to do it automatically.

Reply:
done

Reply:
Not completely, I will show you how to setup VSCode to do it tomorrow